src/views/projects/Browser.svelte

```diff
@@ -156,7 +177,7 @@
             {hash}
             blob={blobResult.blob}
             highlighted={blobResult.highlighted}
-            rawPath={utils.getRawBasePath(project.id, baseUrl, commit)} />
+            rawPath={utils.getRawBasePath(projectId, baseUrl, commit)} />
```

Comment:
I’m not sure whether it’s a good idea to replace the `project` property with the two properties `defaultBranch` and `projectId`. This change shifts the responsibility of unpacking the project object provided by the router from the `Browser` component to the project `View` component. But one of the goals of the routing refactoring is to decouple `View` from the individual view components so that we can change an individual view by only changing the data loading in the router and the component file. In the end we want a simple `View` component that looks much like `App`.

We can tackle this fully at a later stage, but I think it would be good to not move into the opposite direction in the meantime.